app/models/institution_builder.rb

```diff
@@ -841,10 +856,15 @@ def self.build_institution_category_ratings(version_id)
       FROM(
         SELECT
           institution_id,
+          CASE
+            WHEN SUM(rated5_count) + SUM(rated4_count) + SUM(rated3_count)
+              + SUM(rated2_count) + SUM(rated1_count) = 0 THEN NULL::float
```

Comment:
Should == be used here also? Again, I'm rusty on case statements, so I could be wrong.